### DISCUSSION TITLE
HEQUAV G-Port Gimbal support via Serial (mavlink)

### DISCUSSION THREAD

**Discussion by ltwin8** [iNavFlight/inav#10284] [Support]
there are some pretty nice gimbals available from vendors like HEQUAV. They also published some documentation for their gimbal which looks to be of good build quality.

https://www.dropbox.com/scl/fo/fd1aiouf596bpl1aojkev/AJU0DYRcIhTXE8P19toiMeM?rlkey=se46v3wwb3bc0jk4lk6cnvqub&e=1&st=qt1gosge&dl=0

would it be possible to support it in inav via mavlink? that way one could forward specific RC-chanels or chanels mixed with local sources like "stabilized roll" and so on to the gimbal enabling headtracking and mission/attitude specific viewing.

https://store.hequavtech.com/pages/support (if previous link does not work, choose "[G-Port Gimbal Assistant+Manual]")

**Comment by b14ckyy** [comment]
In the works. No ETA yet: https://github.com/iNavFlight/inav/pull/10254 

Moving Ticket to discussion to keep the PR clean and for testing feedback when ready.
